CHANGES

```diff
@@ -8,6 +8,25 @@ Highlights
   well as the number of compute units available on the device.
 - Support for NVIDIA GPUs via a new CUDA backend (currently experimental).
 - Removed support for BBVectorizer
+- LLVM 5.0 support
```

Comment:
Can you make the items full sentences with a capital first letter? Also, did you scan through the log for changes? The version shall be 1.0.